.openshift-ci/begin.sh

```diff
@@ -48,7 +48,7 @@ if [[ "${JOB_NAME:-}" =~ -eks- ]]; then
     set_ci_shared_export USER_ARNS "$(aws sts get-caller-identity | jq -r '.Arn')"
 fi
 
-if [[ "${JOB_NAME:-}" =~ ocp-.*-scanner-v4-tests.* ]]; then
-    info "Setting worker node type for OCP Scanner V4 tests to e2-standard-16"
+if [[ "${JOB_NAME:-}" =~ ocp-.*-scanner-v4-install-tests.* ]]; then
+    info "Setting worker node type for OCP Scanner V4 installation tests to e2-standard-16"
```

Comment:
I need to double-check, but I'm pretty sure this was either specifically for the optional (non-install) Scanner V4 tests or both the non-install and install tests.

Reply:
@mclasmeier do you recall? https://github.com/stackrox/stackrox/pull/11229

Reply:
Sorry, this one slipped through on my side.

Since the resource requirements of Scanner V4 have decreased since the initial integration, I guess this one should be reevaluated. *Maybe* e2-standard-8 could be sufficient.

But, just for some context, I believe this one was put in place due to instability issues in the install test suite. In particular, because the install test suite is doing things like, installing central in one namespace (including scanner v4), installing secured-cluster services in a different namespace (again, including scanner v4), making sure that all deployments come up healthy.

Reply:
@BradLugo I never looked non-installation tests, I believe.

Reply:
since Scanner V4 requires much less (many fewer?) resources, I'll remove this line